src/components/Buttons/Button/Button.scss

```diff
@@ -27,7 +27,7 @@
 
     &:focus,
     &:active {
-        z-index: 100;
+        z-index: 2;
```

Comment:
would 10 here to won't work? if it has to be higher then the autosuggest for some reason, maybe use 20.